test/test_torch.py

```diff
@@ -5198,11 +5198,6 @@ def test_comparison_ops(self):
         for idx in iter_indices(x):
             self.assertEqual(x[idx] >= y[idx], ge[idx] == 1)
 
-    def test_comparison_ops_must_take_bool_output(self):
-        with self.assertRaisesRegex(RuntimeError, 'The output tensor of a comparison or logical op must be a bool'):
```

Comment:
I guess we could remove the check for error message, but keep checking the `out.dtype` is bool tensor in `out = op(a, b)`?

Reply:
I've rewritten this test as outputing bool dtype. Thanks for the review!